### PR TITLE
chore: Implemented mechanism to reference kustomize manifests from github

### PR DESCRIPTION
**Pull request by niallthomson**
#### What this PR does / why we need it:

Currently the Kubernetes manifests applied throughout the labs are cloned to the Cloud9 instance and run directly from the filesystem. This makes it more difficult to update the manifests since the Cloud9 filesystem will need to be regularly updated against the GitHub repository.

This PR changes the content so that it directly references GitHub with kubectl like so:

```
kubectl apply -k github.com/aws-samples/eks-workshop-v2/environment/workspace/modules/security/irsa/service-account?ref=main
```

This means users still do not need to copy and paste large amounts of YAML but it can be updated more efficiently.

A new mechanism has been introduced to the Markdown processing which automatically inserts URLs like the above based on the local path. The maintains a smooth local development experience.

```
kubectl apply -k @{environment/workspace/modules/security/irsa/service-account}
```

#### Which issue(s) this PR fixes:

Fixes #

#### Quality checks

- [ ] My content adheres to the style guidelines
- [ ] I ran `make test` or `make e2e-test` and it was successful

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: terraform/variables.tf [hunk 1]
```diff
@@ -12,7 +12,7 @@ variable "cloud9_owner" {
 
 variable "repository_ref" {
   type        = string
-  default     = "main"
+  default     = "terminal-portability"
```
Comment (svennam92) [inline terraform/variables.tf:15]: does this need to be changed back to main?

File: website/docs/index.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ title: EKS Workshop
 sidebar_position: 1
 ---
 
+asdfasf
```
Comment (svennam92) [inline website/docs/index.md:6]: can we delete this file?

**Comment by svennam92** [comment]
can we add a blurb to the authoring content docs explaining how this works?

**Comment by niallthomson** [comment]
Closing in favor of upcoming rearchitecture PR

**Pull request closed without merging**